Lab.1 patch 1.0

d3nik opened the pull request:
– index.html and folder "images" created;
– source images imported;
– html page filled with content.

d3nik committed:
Lab.1 patch 1.0

– index.html and folder "images" created;
– source images downloaded;
– html page filled with content.

d3nik committed:
Update index.html

– edited document language changed.

vozniuk-lntu commented on index.html:
```diff
@@ -0,0 +1,139 @@
+
+      <ul class="contacts">
+        <li><a href="mailto:info@devstudio.com">info@devstudio.com</a></li>
+        <li><a href="tel:+38 096 111 11 11">+38 096 111 11 11</a></li>
```
прибери пробіли з href="tel:+38 096 111 11 11"

vozniuk-lntu commented on index.html:
```diff
@@ -0,0 +1,139 @@
+      <section class="features">
+        <ul>
+          <li>
+            <h2>УВАГА ДО ДЕТАЛЕЙ</h2>
```
це h3

vozniuk-lntu commented on index.html:
```diff
@@ -0,0 +1,139 @@
+      </section>
+
+      <section class="occupation">
+        <img src="images/working_space_1.jpg" alt="back-end developing" width="15%">
```
приписати висоту і ширину з фігми без пікселів

vozniuk-lntu commented on index.html:
```diff
@@ -0,0 +1,139 @@
+        </ul>
+      </section>
+
+      <section class="occupation">
```
заголовок секцій h2

vozniuk-lntu commented on index.html:
```diff
@@ -0,0 +1,139 @@
+        <img src="images/working_space_3.jpg" alt="front-end developing" width="15%">
+      </section>
+
+      <section class="team">
```
заголовок секцій h2

vozniuk-lntu commented on index.html:
```diff
@@ -0,0 +1,139 @@
+        <figure class="card">
+          <img src="images/Ihor.jpg" alt="Product Designer" width="10%">
+          <figcaption>
+            <h4>Ігор Дем'яненко</h4>
```
h3

vozniuk-lntu commented on index.html:
```diff
@@ -0,0 +1,139 @@
+    </main>
+
+    <footer>
+      <svg
```
СВГ інша у футері

vozniuk-lntu commented on index.html:
```diff
@@ -0,0 +1,139 @@
+      
+      <address>
+        <ul>
+          <li>м. Київ, пр-т Лесі Українки, 26</li>
```
вийми з списка
просто спан

vozniuk-lntu commented on index.html:
```diff
@@ -0,0 +1,139 @@
+        <ul>
+          <li>м. Київ, пр-т Лесі Українки, 26</li>
+          <li><a href="mailto:info@devstudio.com">info@devstudio.com</a></li>
+          <li><a href="tel:+38 096 111 11 11">+38 096 111 11 11</a></li>
```
пробіли прибери

vozniuk-lntu reviewed (approved):
виправиш правки в лабі 2

d3nik committed:
patch 1.1 –  Some fixes

– deleted spaces in href="tel:";
– fixed some headings level (from h2 to h3 etc);
– added section names;
– added widths and heights from Figma project;
– changed svg in footer;
– some fixes in footer;

d3nik committed:
Merge branch 'lab-1' of https://github.com/d3nik/Serediuk-Denys-IPZ-34 into lab-1
